webapi/admin.go

```diff
@@ -197,12 +198,24 @@ func ticketSearch(c *gin.Context) {
 	})
 }
 
+//checkPasswordHash compare hash value of given AdminPass with AdminPass hash.
```

Comment:
Comment makes references to `AdminPass` but I don't see any such variable in this function. Also, look at the documentation for `bcrypt.CompareHashAndPassword` to confirm what it does, I'm not sure the function compares hash values because only one of the 2 parameters being passed is a hashed value.